index.md

```diff
@@ -264,6 +295,15 @@ Web Content Accessibility Guidelines
 
 
 
+---
+<!-- .slide: data-background="images/background/01.svg" -->
+
+<p class="centerp">
+    <a href="http://servicesbeta.esri.com/demos/a11y/index.html">Demo: Automated test using aXe</a>
```

Comment:
For this one I added the demo code to this repo in the demos folder demos/map-demo/index.html so maybe link to that version since it's been updated to use the latest version of the JSAPI